main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+
+	"github.com/apache/beam/sdks/v2/go/pkg/beam"
+	"github.com/apache/beam/sdks/v2/go/pkg/beam/x/beamx"
+)
+
+var (
+	input_text = flag.String("input-text", "Default input text", "Input text to print.")
+)
+
+func init() {
+	// DoFns should be registered with Beam.
+	beam.RegisterFunction(printAndEmit)
+}
+
+func printAndEmit(element string, emit func(string)) {
+	fmt.Println(element)
+	emit(element)
+}
+
+func my_pipeline(scope beam.Scope, input_text string) beam.PCollection {
```

Comment:
```suggestion
func myPipeline(scope beam.Scope, input_text string) beam.PCollection {
```
Go style has methods/functions use camelcase rather than snake case (snake case is for flags).